kuma/banners/jinja2/banners/includes/developer-needs.html

```diff
@@ -0,0 +1,15 @@
+<div id="developer-needs" class="mdn-cta-content-container">
+    <h2 class="mdn-cta-title">
+        {{ _('MDN Developer Survey') }}
+    </h2>
+    <p class="mdn-cta-copy">
+        {{ _('We want to know learn more about the needs of developers') }}
+    </p>
+</div>
+<p class="mdn-cta-button-container">
+    <a
+        href="https://developer.mozilla.org/"
```

Comment:
Is the survey link not ready yet? Is this where it will go when it is ready?

Reply:
Noy yet and yes ;)